src/components/SistentNavigation/index.js

```diff
@@ -114,13 +117,70 @@ const TOC = () => {
             </div>
           </li>
           <li>
-            <Link
-              to="/projects/sistent/components"
-              activeClassName="active"
-              className="toc-sub-heading toc-sub-inline"
-            >
-              Components
-            </Link>
+            <div>
+              <li
+                className="components"
+                onClick={() => setExpandComponents((prev) => !prev)}
+              >
+                <Link
+                  to="/projects/sistent/components"
+                  className= "components-heading"
+                  activeClassName="active"
+                >
+                  Components
+                </Link>
+                {expandComponents ? <IoIosArrowUp /> : <IoIosArrowDown />}
+              </li>
+              {expandComponents && (
+                <div className="element-sublinks">
+                  <li>
+                    <Link
+                      to="/projects/sistent/components/button"
+                      className={`toc-sub-heading toc-sub-inline element-item ${
+                        location.pathname.includes(
+                          "/projects/sistent/components/button"
+                        )
+                          ? "active"
+                          : ""
+                      }`}
+                      activeClassName="active"
+                    >
+                      Button
```

Comment:
why they are static? Shouldn't they be dynamic

Reply:
Just followed the previous way of code (i.e. static). Dynamic would be great. I'll look into it.

Reply:
thanks, please propose a change if you would

Reply:
Yes, I will.